composer.json

```diff
@@ -1,13 +1,13 @@
 {
     "name": "freshbitsweb/laratables",
-    "description": "Ajax support of DataTables for Laravel 5.5+",
+    "description": "Ajax support of DataTables for Laravel",
     "require": {
-        "php": "^7.3|^7.4|^8.0",
-        "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0|^7.0|^8.0|^9.0"
+        "php": "^8.0",
+        "illuminate/support": "^9.33.0|^10.0"
```

Comment:
L9.33 is required, because that's when they added PHP8.2 support to L9

Reply:
@SanderMuller But they should still be able to use Laravel 9 with PHP 8.0 or PHP 8.1, right?

In that case, can we keep `"illuminate/support": "^9.0|^10.0"`?

Reply:
That would make the P8.2 - L9.* - prefer-lowest check fail.

Reply:
Not that I know of, tagging a major release is probably the best option anyway. I think 9.33 should not be a big hurdle for 9.* users since they should upgrade for security patches anyway.